DFC.Api.Content/StartUp/FunctionStartupExtension.cs

```diff
@@ -0,0 +1,40 @@
+﻿using DFC.ServiceTaxonomy.ApiFunction.Models;
+using DFC.ServiceTaxonomy.ApiFunction.StartUp;
+using DFC.ServiceTaxonomy.Neo4j.Configuration;
+using DFC.ServiceTaxonomy.Neo4j.Log;
+using DFC.ServiceTaxonomy.Neo4j.Services;
+using Microsoft.Azure.Functions.Extensions.DependencyInjection;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Neo4j.Driver;
+using System.IO;
+
+[assembly: FunctionsStartup(typeof(FunctionStartupExtension))]
+
+namespace DFC.ServiceTaxonomy.ApiFunction.StartUp
+{
+    public class FunctionStartupExtension : FunctionsStartup
+    {
+        public override void Configure(IFunctionsHostBuilder builder)
+        {
+            var config = new ConfigurationBuilder()
+                .SetBasePath(Directory.GetCurrentDirectory())
+                .AddJsonFile("appsettings.json", optional: false, reloadOnChange: true)
+                .AddEnvironmentVariables()
```

Comment:
It'd be good to add an environment specific appsettings too, or just an optional hard-coded "appsettings.Development.json".